### ISSUE TITLE
Proposal: support messageformat

### ISSUE THREAD

**Issue by fxck** [ngneat/transloco#11]
https://messageformat.github.io/messageformat/

https://github.com/lephyrus/ngx-translate-messageformat-compiler

**Comment by shaharkazaz** [comment]
@fxck We are planning to start work on it next week.
You are welcome to take this task and open a PR if you want 🙂

**Comment by Coly010** [comment]
@shaharkazaz I am going to try work on implementing this.

**Comment by shaharkazaz** [comment]
@Coly010 go for it 🙂

**Comment by Coly010** [comment]
@shaharkazaz Can you clarify a potential requirement relating to this Proposal.

Should the transpiler accept both `{{}} and MessageFormat`:

`{{t.translation | translocoParams: { foo: 1, bar: 'baz' } }}`

where `translation` is: 
`'translation': 'My number {{foo}}! My {bar, select, baz {world} foobar {car} other {something}}!'`

which will transpile to:
`My number 1! My world!`

OR should it only accept MessageFormat?

My current implementation requires the user to alter the providers array in `app.module.ts` to inject a different `TRANSPILER`:

```ts
{
    provide: TRANSLOCO_TRANSPILER,
    useClass: MessageFormatTranspiler
}
```

**Comment by shaharkazaz** [comment]
Fixed in #15
